common/scripts/poll-npm-package-published.mjs

```diff
@@ -0,0 +1,57 @@
+import { get } from "https";
+import process from "process";
+
+/**
+ * Script that polls the npm registry to see if a @azure/communication-react package version published.
+ * By default, it polls on short intervals (5 seconds) and timesout after a longer interval (5 minutes).
+ *
+ * @params {string} version - the version to check for
+ *
+ * @example node ./poll-npm-package-published.js 1.3.0
+ */
+
+const POLL_INTERVAL_MS = 5000; // 5 seconds in millis
+const TIMEOUT_TIME_MS = 12 * 1000; // 5 minutes in millis
+const GET_REQUEST_TIMEOUT_MS = 5000;
+
+const GET_REQUEST_OPTIONS = {
+  hostname: 'registry.npmjs.org',
+  timeout: GET_REQUEST_TIMEOUT_MS
+}
+
+const checkNpm = (packageName, packageVersion) => {
+  return new Promise((resolve, reject) => {
+    const request = get({...GET_REQUEST_OPTIONS, path: `/${packageName}/${packageVersion}`}, (response) => {
```

Comment:
Do you need to URL encode the package name and/or version?

Reply:
Seemingly not (works on the command line), but will know once the nightly cd runs with this